### ISSUE TITLE
Problem("Utf8Error { valid_up_to: 8841, error_len: Some(1) }")

### ISSUE THREAD

**Issue by k86td** [dmadisetti/steam-tui#70]
I get this while trying to log in to steam-tui on a NixOS machine:

```
Problem("Utf8Error { valid_up_to: 8841, error_len: Some(1) }")
```

**Comment by dmadisetti** [comment]
Using nixpkgs or the flake?

**Comment by k86td** [comment]
Nixpkgs, I've tried stable and unstable channels.

I didn't put the time to clone the repo and debug on my end yet.

**Comment by dmadisetti** [comment]
NixPkgs is way behind, I was hoping to cut another release after #47, but I haven't worked on this in a bit

**Comment by Titaniumtown** [comment]
I am able to reproduce this with nixpkgs-unstable

**Comment by dmadisetti** [comment]
Will close on nix release

**Comment by laycookie** [comment]
Any updates on this issue?
Also I assume I can fix it by using the flake but Im not sure how to set it up can anyone help me here?

I added steam-tui.url to inputs but Im not sure what to do next.

**Comment by dmadisetti** [comment]
Woops. Thanks for the reminder

**Comment by dmadisetti** [comment]
nixpkgs is updated. Tested locally. Please reopen if another issue arises

(It's on head, will be in unstable after a few days on cache)
